spec/podfile_spec.rb

```diff
@@ -114,6 +109,16 @@ module Pod
         Podfile.new { set_arc_compatibility_flag! }.should.set_arc_compatibility_flag
       end
 
+      it 'returns the installation method' do
+        name, options = Podfile.new {}.installation_method
+        name.should == 'cocoapods'
+        options.should == {}
```

Comment:
I like this format ( https://github.com/artsy/eigen/pull/1003/files ) - I think it's good and future-proof :+1: